packages/features/webhooks/lib/handleWebhookScheduledTriggers.ts

```diff
@@ -56,13 +56,23 @@ export async function handleWebhookScheduledTriggers(prisma: PrismaClient) {
         !job.payload || jsonParse(job.payload) ? "application/json" : "application/x-www-form-urlencoded",
     };
 
+    const parsedPayload = jsonParse(job.payload);
+
+    const body = JSON.stringify({
+      ...(job.payload && parsedPayload),
+      triggerEvent: parsedPayload.triggerEvent,
+      payload: parsedPayload,
```

Comment:
payload still has different fields than for the `Booking created` webhook and as it's described in the docs 

We need to make sure to already save the correct payload on creation of `WebhookScheduledTriggers`. We are sending different data than for the other triggers